sidekiq-limit_fetch.gemspec

```diff
@@ -2,11 +2,13 @@ Gem::Specification.new do |gem|
   gem.name          = 'sidekiq-limit_fetch'
   gem.version       = '3.4.0'
```

Comment:
Did you want to bump this?

Reply:
Did that in a follow-up PR where I also add the tag, just so the tag is only tied to the version bump https://github.com/RCGTConsulting/sidekiq-limit_fetch/pull/2